resources/dex/templates/dex-users-secret.yaml

```diff
@@ -72,3 +72,21 @@ data:
   # developer-user
   username: ZGV2ZWxvcGVyLXVzZXI=
   password: {{ randAlphaNum 12 | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name:  test-namespace-admin-user
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook: "pre-install"
+  labels:
+    "dex-user-config": "true"
+    "user-email": "namespace.admin.kyma.cx"
+data:
+  # namespace.admin@kyma.cx
+  email: bmFtZXNwYWNlLmFkbWluQGt5bWEuY3g=
+  # namespace-admin-user
+  username: bmFtZXNwYWNlLWFkbWluLXVzZXI=
```

Comment:
Do we really use this value somewhere?

Reply:
Yes. In a dex configmap. Like in this example: https://github.com/dexidp/dex/blob/master/examples/config-dev.yaml#L94